cfme/tests/networks/test_sdn_ports.py

```diff
@@ -44,6 +45,7 @@ def test_sdn_port_detail_name(provider, appliance):
             pass
 
 
+@pytest.mark.meta(blockers=[BZ(1718495, forced_streams=['5.11'])])
```

Comment:
@mmojzis I see that this BZ is verified in CFME 5.11, why is `forced_streams` necessary here?

Reply:
@john-dupuy Thanks for pointing that out. I tested it with already verified BZ and then forgot to change to correct BZ. It should be okay now.